cases/file_to_s3_reliability/configurations/default/ansible.yml

```diff
@@ -0,0 +1,2 @@
+---
+foo: "bar"
```

Comment:
It didn't let me not have this file.

Presumably, we could use this for things like the bucket name but a few things weren't clear:

1. How to template into the vector toml while leaving templated fields alone
2. How to get this same variable into terraform

Reply:
We do the templating at part of `bin/test` and our `lib` facility, and pass it to both ansible and terraform!